packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -393,10 +419,10 @@ const Conversation = WebexPlugin.extend({
    * provisional activity
    * @returns {Promise<Activity>}
    */
-  async leave(conversation, participant, activity) {
+  leave(conversation, participant, activity) {
```

Comment:
Glad we are able to remove the async modifiers to help clean up code readability.